agent/instructions.md

```
@@ -0,0 +1,16 @@
+You are Hugo Richard's personal GitHub agent.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)